src/step-3-logging/index.js

```diff
@@ -0,0 +1,15 @@
+import Fastify from 'fastify'
+
+function buildServer() {
+  const fastify = Fastify({
+    logger: true,
```

Comment:
We should cover the prettyPrint option and mention that it should only be used for development.